Neos.ContentRepository.Core/Classes/NodeType/ConstraintCheck.php

```diff
@@ -14,6 +14,15 @@
     private function __construct(
         private array $constraints
     ) {
+        foreach ($this->constraints as $constraint => $value) {
```

Comment:
Shouldn't we build a `NodeTypes` collection similar to `Projections` and use that as argument here?

Reply:
i would argue the `ConstraintCheck` is already this collection

Reply:
Right, probably true, I guess I expected a "fromArray" or somthing.